compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -974,7 +973,6 @@ static TR::InstOpCode::Mnemonic cmp2branch(TR::ILOpCodes op, TR::CodeGenerator *
        case TR::lucmpne:
        case TR::fcmpne:
        case TR::dcmpne:
-       case TR::fcmpneu:
```

Comment:
Why is this being removed?

Reply:
This should not be removed. `fcmpneu` is not even on the Removal Table. Wondering why did it still pass all the tests.